at_lookup/lib/src/util/secure_socket_util.dart

```diff
@@ -18,10 +21,12 @@ class SecureSocketUtil {
           throw AtException(
               'decryptPackets set to true but path to trusted certificated not provided');
         }
-        return await SecureSocket.connect(host, int.parse(port),
+        _secureSocket = await SecureSocket.connect(host, int.parse(port),
             context: securityContext,
             keyLog: (line) =>
                 keysFile.writeAsStringSync(line, mode: FileMode.append));
+        _secureSocket.setOption(SocketOption.tcpNoDelay, true);
```

Comment:
Probably you have to do set option before connect

Reply:
That might not be possible sir. SecureSocket class is abstract and secureSocket.connect() is a way to instantiate a secureSocket object. We can only set options after we have a secureSocket object so this is the possible way.